fnet/network_test.go

```diff
@@ -169,6 +169,40 @@ func TestResolveIpV6(t *testing.T) {
 	}
 }
 
+func TestJoinHostAndPort(t *testing.T) {
+	var tests = []struct {
+		inputPort string
+		addr      *net.TCPAddr
+		expected  string
+	}{
+		{"8080", &net.TCPAddr{
+			IP:   []byte{192, 168, 2, 3},
+			Port: 8081,
+		}, "192.168.2.3:8081"},
```

Comment:
this would be localhost unless you put 192.168.2.3:8081 as input (the tests need to include using addr:port)